corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -996,6 +996,59 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
     VideoEntry.prototype = Object.create(FileEntry.prototype);
     VideoEntry.prototype.constructor = FileEntry;
 
+    /**
+     * Represents a signature, which requires the user to draw a signature.
```

Comment:
Sounds funny. Maybe "Represents a widget"?

Reply:
I agree this sounds weird. How's just: "Represents a signature entry, which requires the user to draw a signature." ?